.sauce/cypress.yml

```diff
@@ -36,3 +36,7 @@ artifacts:
     match:
       - console.log
     directory: ./artifacts/
+notifications:
+  slack:
+    channels: ["some channel1", "some channel2"]
```

Comment:
Since the CI will execute those configuration, shouldn't we use actual channels where we can receive those notifications ?

Reply:
Aye, I second this one. These config files are for the pipeline, so they should have sensible values as well 💡

Reply:
@FriggaHel I added `devx-slack-notifications` channel for all configs. After deploying slack app I can add this app to `devx-slack-notifications` channel.  Regarding env vars, I would skip it for the first iteration and add it later on.